frontend/src/scenes/error-tracking/queries.ts

```diff
@@ -131,6 +135,8 @@ export const errorTrackingGroupQuery = ({
         dateRange: dateRange,
         filterGroup: filterGroup as PropertyGroupFilter,
         filterTestAccounts: filterTestAccounts,
+        limit: limit,
```

Comment:
can you add these params to a backend test as well so we can see the snapshot update? iirc there's no order by clause, which means the offset can be arbitrary and non-deterministic, which can screw up our stats (example: same event is counted multiple times in different offsets, and some other event is completely discarded)

Reply:
> there's no default order by clause, which means the offset can be arbitrary and non-deterministic

yeah pagination has to be deterministic